packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -59,58 +58,118 @@ const NominationBanner = (props) => {
       (declineAppModalVisible) => !declineAppModalVisible
     );
   };
-
+  function attachmentsHandled() {
+    const value = false;
+    activeNomination.attachments = value;
```

Comment:
Can we call this variable `hasAttachments` instead? Calling it `attachments` makes it sound like it's an array of attachments instead of a boolean indicating whether it has attachments or not.

Reply:
changed